GothicLlamasAPI/src/test/java/com/kainos/ea/resources/WebControllerTest.java

```diff
@@ -28,6 +28,8 @@ public void setup() {
         wc = new WebController(wcService);
     }
 
+    //US001 test cases
```

Comment:
Comments like these may quickly become _wrong_, as the production functionality changes and the tests are accordingly updated. Not recommended